app/models/installment.rb

```diff
@@ -0,0 +1,15 @@
+class Installment < ApplicationRecord
+  validates :off, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 100 }, presence: true
+  validates :value, numericality: { greater_than_or_equal_to: 0 }, presence: true
+  validates :total_value, numericality: { greater_than_or_equal_to: 0 }, presence: true
+  validates :payment_date, presence: true
+  validates :due_date, presence: true
+
+  before_validation :calculate_total_value
+
+  private
+
+    def calculate_total_value
```

Comment:
[Indent the public, protected, and private methods as much the method definitions they apply to.](https://github.com/airbnb/ruby#access-modifiers)

Reply:
After check, it broke in rubocop and CI.

Reply:
I saw. You'are using `rubocop-rails_config` gem to use the `rails guidelines`. Isn't a big deal.
We use the standard rubocop guidelines here though.